### ISSUE TITLE
SIMBA VALUE ERROR- Issue Importing H5 and slp tracking data 

### ISSUE THREAD

**Issue by wallawhitecoat** [sgoldenlab/simba#365]
![Simba Problem](https://github.com/sgoldenlab/simba/assets/169295221/57a868cd-de39-4b50-87ee-56e69cbcea1b)
![SimBA Problem 2](https://github.com/sgoldenlab/simba/assets/169295221/5e61e32f-97bb-4c7e-b29f-8b20ceae86e9)

I'm trying to start a new project and import labels from sleap (resident intruder). I tried importing as a .slp file and a H5 file I had. Neither worked and I get this message from SimBA, or the GUI quits. I only imported 1 video, ten mins long, 60fps, and over 36,000 frames. Any help would be appreciated, thank you! 

- Windows 10, version 22H2
- Python Version [3.10.14]
- using miniconda3 Version [24.4.0]

**Comment by sronilsson** [comment]
Hi @wallawhitecoat thanks for reporting - this error happens as SimBA looks for the body-parts from SLEAP that are most proximal to your button clicks, in order to assign the animals their correct identities across videos. 

Looking at the image of the resident and Intruder above, your SLEAP tracking does not appear to contain any data for the frame being shown. If there is no data (and you perform no interpolation), SimBA places the body-parts at coordinates 0,0 - top left of the image: 

<img width="666" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/88b274ba-cc57-4772-9a73-47be97574e4a">

You could try to perform interpolation as you import the data into SimBA to get body-part predictions for all frames. Alternatively is to improve the pose-estimation model so it generates accurate predictions for all frames and no interpolation is needed. 

Let me know if that makes sense!

Simon

**Comment by wallawhitecoat** [comment]
Hi Simon, thank you for the response. I was able to export an H5 file with predictions on every frame. Here is what my skeleton looks like in SimBA and in SLEAP. 
![Skeleton settings](https://github.com/sgoldenlab/simba/assets/169295221/e6b11fd2-6397-437b-8834-b8032723ec50)
![Project Config SimBA](https://github.com/sgoldenlab/simba/assets/169295221/c87ea471-b5ba-48d3-b681-bbbc8ec898d0)
![SLEAP Skeleton](https://github.com/sgoldenlab/simba/assets/169295221/8f80ff14-710d-4205-aadc-45607af768c6)

Unfortunately, I keep getting this notification from SimBA, what should I do? Thank you!
![SimBA Problem3](https://github.com/sgoldenlab/simba/assets/169295221/2ab73554-b6e4-4b8b-b775-65f575a1cf16)

**Comment by sronilsson** [comment]
Hi @wallawhitecoat ! 

Can you drop the `labels.v0012.h5` in this thread and I can take a look what is going on in there? You should be able to zip the file up and drag it into the thread. I am not sure why SimBA thinks there are only 14 body-parts in there.

**Comment by wallawhitecoat** [comment]
Sure, let me know if this file works! 

[labels.v0012.zip](https://github.com/user-attachments/files/15595702/labels.v0012.zip)

**Comment by sronilsson** [comment]
Thanks - just a sanity check - the file `labels.v0012.h5` is only 93kB, very small, and as it's called "labels", I have to ask: 

Does this file contain your hand annotations SLEAP ? Or do the files contain machine generated body-part predictions for all frames in the respective videos? I ask as previously [HERE](https://github.com/sgoldenlab/simba/issues/339) and more specifically [HERE](https://github.com/sgoldenlab/simba/issues/339#issuecomment-1972002345) someone also tried to use a file with the same filenames as yours and I think it was just hand labels and not machine learning predictions.

**Comment by wallawhitecoat** [comment]
It contains both hand annotations and machine generated body-part predictions for all frames. I went back to sleap and predicted all of the frames on a copy of the video I was using and exported that prediction as an H5 file. The new file is 4,094KB.
Now I get a similar problem as before, but instead of saying that the number of body parts is 11 instead of 22. 
![image](https://github.com/sgoldenlab/simba/assets/169295221/1aa26024-a589-4ed7-985b-12902719680d)
[labels.v0013.zip](https://github.com/user-attachments/files/15596814/labels.v0013.zip)

This is how I'm saving in SLEAP
![image](https://github.com/sgoldenlab/simba/assets/169295221/aec3e185-4070-4bad-a50d-66c456da28ba)

**Comment by wallawhitecoat** [comment]
UPDATE: I was able to download a csv file, but it only shows tracking on one mouse 
![image](https://github.com/sgoldenlab/simba/assets/169295221/9776dd7c-d059-4189-8f86-af5eb3389d38)

The CSV file looks like this when I open it, I notice that there is only one "instance score" for each frame, if that is helpful in determining why only 11 nodes are being recognized. 
![image](https://github.com/sgoldenlab/simba/assets/169295221/37228eb4-a2af-46af-9373-1179c7d252dd)

**Comment by sronilsson** [comment]
Yeah that seems likely - is the `track` field all blank? Even if you sort by `frame_idx` field? That would explain why there are only have the expected body-parts. 

In the `frame_idx` field, are there two instances of every frame number?

**Comment by wallawhitecoat** [comment]
That was it, thank you! I re ran my trained model and included tracks on sleap, so all of my predictions have tracks now. 

![image](https://github.com/sgoldenlab/simba/assets/169295221/e29853a2-03c2-4685-95b7-8c4f99c94cdc)
![image](https://github.com/sgoldenlab/simba/assets/169295221/56e37fde-edd8-4a1f-aa9f-f9b6c0bdb2e0)

**Comment by sronilsson** [comment]
Great news, onwards and upwards! :) let me know if anything else comes up
